components/SwipeListView.js

```diff
@@ -183,7 +183,9 @@ class SwipeListView extends PureComponent {
                     }
                     ref={row => (this._rows[key] = row)}
                     swipeGestureBegan={() => this.rowSwipeGestureBegan(key)}
-                    swipeGestureEnded={() => this.rowSwipeGestureEnded(key)}
+                    swipeGestureEnded={(swipeKey, data) =>
+                        this.rowSwipeGestureEnded(key, data)
```

Comment:
Should this line read `this.rowSwipeGestureEnded(swipeKey, data)` ?

Reply:
A good question. If I understood correctly, the `swipeKey` is the prop passed to an individual `SwipeRow` whereas the `key` is passed from the `SwipeListView` renderItem call on line 346.

I passed the `swipeKey` up from the `SwipeRow` for cases when it is being used individually outside the context of a `SwipeListView`. Does that make sense? I can revert if I'm misunderstanding.

Reply:
Ah okay. I think I understand this now. So in this case you actually don't care about the `swipeKey` since you'll be passing the `key`. So maybe it would make more sense as:

```
swipeGestureEnded={(_, data) =>
    this.rowSwipeGestureEnded(key, data)
```

What do you think?

Reply:
Definitely more clear. I’ll push a new commit shortly 👍🏼.